types/Config.js

```diff
@@ -15,7 +15,7 @@ export type HasteConfig = {|
   defaultPlatform?: ?string,
   hasteImplModulePath?: string,
   platforms?: Array<string>,
-  providesModuleNodeModules: Array<string>,
+  providesModuleNodeModules: Array<string>
```

Comment:
Yeah, I was about to add that comment... this is what my Prettier config is formatting this file to... I'll double check if I have something wrong on my setup